src/components/error-summary/error-summary.njk

```diff
@@ -1,19 +1,20 @@
 {% from "error-summary/macro.njk" import govukErrorSummary %}
 
-{{- govukErrorSummary(
-  classes='',
-  title='Message to alert the user to a problem goes here',
-  description='Optional description of the errors and how to correct them',
-  listClasses='',
-  listItems=[
-    {
-      text: 'Descriptive link to the question with an error',
-      url: '#example-error-1'
-    },
-    {
-      text: 'Descriptive link to the question with an error',
-      url: '#example-error-2'
-    }
-  ]
-)
--}}
+{{ govukErrorSummary({
+  "title": "Message to alert the user to a problem goes here",
+  "description": "Optional description of the errors and how to correct them",
+  "listOfErrors": {
+    "classes": "govuk-c-error-summary__list",
```

Comment:
I don't think users should have to remember to add this class here - it needs to be merged into the params when the list is included.

Reply:
now resolved with the updated list component merged in

Reply:
Can you update this example then please?